sdk/communication/communication-phone-numbers/CHANGELOG.md

```diff
@@ -1,27 +1,21 @@
 # Release History
 
-## 1.2.0-beta.5 (Unreleased)
+## 1.2.0-beta.4 (2023-01-10)
+- Adds support for Azure Communication Services Phone Numbers Browse API Methods.
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+- Added support for API version `2022-12-01`, giving users the ability to: 
+  - Get all supported countries
+  - Get all supported localities given a country code.
+  - Get all Toll-Free area codes from a given country code.
+  - Get all Geographic area codes from a given country code / locality.
+  - Get all offerings from a given country code.
 
```

Comment:
Please add line for SIP routing client:
- Added new SIP routing client for handling Direct routing numbers.